core/tests/test_chunk.h

```diff
@@ -0,0 +1,156 @@
+// -------------------------------------------------------------
+//  Cubzh Core Unit Tests
+//  test_chunk.h
+//  Created by Nino PLANE on October 31, 2022.
+// -------------------------------------------------------------
+
+#pragma once
+
+#include "block.h"
+#include "chunk.h"
+#include "int3.h"
+
+///// Some function are left untested :
+// --- chunk_get_neighbor()
+// --- chunk_leave_neighborhood()
+// --- chunk_move_in_neighborhood()
+// --- chunk_get_vbma()
+// --- chunk_set_vbma()
+// --- chunk_write_vertices()
+/////
+
+// Create a chunk and check if the default values are the one used
+// Also check all of these function :
+// --- chunk_new()
+// --- chunk_get_pos()
+// --- chunk_get_nb_blocks()
+//////
+void test_chunk_new(void) {
+    Chunk *a = chunk_new(10, 10, 10);
```

Comment:
A better variable name could be find for the chunk here, like c or chunk.
Also, it's probably best to use different values for the 3 dimensions x, y, z.